packages/grafana-data/rollup.config.ts

```diff
@@ -16,18 +16,4 @@ export default [
     plugins,
     output: [cjsOutput(pkg), esmOutput(pkg, 'grafana-data')],
   },
-  tsDeclarationOutput(pkg),
-  tsDeclarationOutput(pkg, {
```

Comment:
@jackw  - won't this have any impact on plugins? Also - what's the `unstable` output?

Reply:
> won't this have any impact on plugins? Also - what's the unstable output?

I don't think it will but I need to publish locally to double check this first. I've moved the types references to point to the files tsc generates. The `tsDeclarationOutput` function took those tsc generated declaration files and rolled them up into single files. The unstable file is required for the [unstable entrypoint](https://github.com/grafana/grafana/blob/main/packages/grafana-data/package.json#L29-L32). [Slack conversation here](https://raintank-corp.slack.com/archives/CHKLEJ668/p1743072262483289).